src/TopFilters/index.js

```diff
@@ -6,117 +6,122 @@
 /* eslint-disable react/prop-types */
 /* eslint-disable react/prefer-stateless-function */
 
-import React, { Component } from 'react';
+import React from 'react';
 import clsx from 'clsx';
-import { Switch, FormControlLabel } from '@material-ui/core';
+import { Switch, FormControlLabel, makeStyles } from '@material-ui/core';
 import styles from '../global-sass-files/_grid.scss';
 import filterStyles from './top-filters.scss';
 import Wrapper from '../Wrapper';
 import SearchLive from '../Search/SearchLive';
 import Button from '../Button';
 
-class TopFilters extends Component {
-  render() {
-    const { fullText, switches, onChange } = this.props;
+const useStyles = makeStyles({
+  labelFontSize: {
+    fontSize: '13px',
+  },
+});
+const TopFilters = (props) => {
```

Comment:
You can spread props like this: 
```suggestion
const TopFilters = ({ fullText, switches, onChange }) => {
```